src/app/header/header.component.ts

```diff
@@ -20,40 +17,36 @@ enum RoleLinks{
 })
 export class HeaderComponent implements OnInit {
 
-  role:string="";
-  roles=RoleLinks;
-  
+  user = {
+    firstName: 'Іванов В. М'
+  }
+  showModalReg: boolean = false;
+
   @Select(UserRegistrationState.isAuthorized)
   isAuthorized$: Observable<boolean>;
-  @Select(UserRegistrationState.userName)
-  userName$: Observable<string>;
-  @Select(UserRegistrationState.role)
-  role$: Observable<string>;
-  
+
   constructor(private modalDialog: MatDialog,
-              public store: Store,
-              private actions$: Actions,
-              public snackBar: MatSnackBar,
-              private router: Router) {
+    public store: Store,
+    private actions$: Actions,
+    public snackBar: MatSnackBar,
+    private router: Router) { 
       actions$.pipe(
         ofAction(AuthFail)
       ).subscribe(action => {
           this.snackBar.open('Check your connection', 'Try again!', {
           duration: 5000,
           panelClass: ['red-snackbar'],
           });
-      });
+      })
     }
+
   ngOnInit(): void {
-    this.store.dispatch(new CheckAuth());
-    this.role$.subscribe(()=>{
-      this.role = this.store.selectSnapshot<string>(UserRegistrationState.role);
-    })
+    this.store.dispatch(new CheckAuth())
   }
   logout(): void {
-    this.store.dispatch(new Logout());
+    this.store.dispatch(new Logout())  
```

Comment:
i believe semicolon is needed

Reply:
I didn't do it manually, when I've pushed changes to git, it automatically removes this semicolon